index.html

```diff
@@ -4975,6 +4975,634 @@ <h2 id="AudioWorkerParamDescriptor">
           </section>
         </section>
       </section>
+      <section>
+        <h2 id="AudioWorklet">
+          The <dfn>AudioWorklet</dfn> interface
+        </h2>
+        <p>
+          The <code>AudioWorklet</code> object allows importing user-supplied
+          Javascript code that runs on the audio <a>rendering thread</a>. This
+          processing mechanism ensures the synchronous execution of the script
+          code with other built-in <code>AudioNode</code>s in the audio graph.
+        </p>
+        <p>
+          The <code>AudioWorklet</code> object stores the definitions of a
+          custom audio processor and an <code>AudioWorkletNode</code> object
```

Comment:
Consistent pluralization: "...of custom audio processors and AudioWorkletNodes can be constructed based on these definitions."